1.0/articles/es6.md

```diff
@@ -0,0 +1,298 @@
+2. Use the `beforeRegister` method instead of `createdCallback`. This is a special method
+Polymer calls before creating your element and is where you should define your `is` property (element's name) and [`properties` object](../docs/devguide/properties.html).
+3. When registering an element, use the `Polymer()` constructor instead of `document.registerElement`.
+
+Here's our stock ticker:
+
+    <link rel="import" href="../polymer/polymer.html">
+
+    <dom-module id="stock-ticker">
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+    <template>
+      ...
+    </template>
+    <script>
+    (function() {
+      'use strict';
+
+      class StockTicker {
+
+        // Element setup goes in beforeRegister instead of createdCallback.
+        beforeRegister() {
+          this.is = 'stock-ticker';
+
+          // Define the properties object in beforeRegister.
+          this.properties = {
+            symbols: {
+              type: Array,
+              value: function() { return []; },
+              observer: '_updateQuotes'
+            }
+          };
+        }
+
+        // Define other lifecycle methods as you need.
+        ready() { ... }
+        attached() { ... }
+        detached() { ... }
+        attributeChanged() { ... }
+
+        _updateQuotes() {
+          // Same as the vanilla component.
+        }
+      }
+
+      // Register the element using Polymer's constructor.
+      Polymer(StockTicker);
+    })();
+    </script>
+    </dom-module>
+
+<p layout horizontal center-center>
+<a href="http://ebidel.github.io/polymer-experiments/polymersummit/stockticker/" target="_blank">
+  <paper-button raised class="blue">Stock Ticker Demo</paper-button>
+</a>
+<a href="https://github.com/ebidel/polymer-experiments/tree/master/polymersummit/stockticker" target="_blank">
+  <paper-button raised class="blue">Full source</paper-button>
+</a>
+</p>
+
+## Building for production
+
+**Note** ES6 classes currently work in Chrome 42+, Safari 9+, Edge preview, and FF nightly.
+{: .alert .alert-info }
+
+For our ES6 code to work in all modern browsers, we need to transpile it to an ES5 equivalent. [BabelJS](https://babeljs.io/) is my personal favorite. It's the most popular at the moment and comes with
+a convenient CLI and Gulp/Grunt/Browserify workflows.
+
+To support all browsers, follow these steps:
+
+1. [Vulcanize](concatenating-web-components.html) your element(s) by inlining the script and CSS:
+
+        vulcanize element.html --inline-script --inline-css > element.v.html
+
+2. Pipe the output to [cripser](https://www.npmjs.com/package/gulp-crisper). Crisper is a tool that extracts all inline `<script>` and creates a standalone .js file.
+
+        crisper -s element.v.html -h element.v.html -j element.js
+
+3. Run the standalone JS file through Babel:
+
+        babel element.js -o element.js
+
+   **Note**: I'm overwriting the original source with Babel's output so we don't have to
```

Comment:
Maybe make clear that by "original" you mean, "the JavaScript source file created by crisper."